permutohedral_lattice/build.sh

```diff
@@ -5,9 +5,9 @@ mkdir build_dir
 cd build_dir
 
 
-CUDA_COMPILER=/N/soft/rhel7/cuda/10.2/bin/nvcc
-CXX_COMPILER=/N/soft/rhel7/gcc/8.4.0/bin/g++
-CUDA_INCLUDE=/N/soft/rhel7/cuda/10.2/include
+CUDA_COMPILER=/N/soft/sles15/cuda/11.7/bin/nvcc
```

Comment:
Aren't these paths strictly HPC? We should either have a relative path or detected path that works for all users, or indicate (with general descriptions on how) that they need to find these paths in the readme file

Reply:
Yup, the initial code that was present had it with respect to HPC. Hence, I had kept it in a similar fashion. But I can replace it with "path_to_nvcc" and stuff like that if required. I'll replace them so that it is more generic.